MCprep_addon/spawner/mcmodel.py

```diff
@@ -20,16 +20,25 @@
 import json
 from mathutils import Vector
 from math import sin, cos, radians
+from pathlib import Path
 
 import bpy
 import bmesh
+from bpy.types import (
+  Context, Object, Material
+)
```

Comment:
Again as suggested earlier, ditch the use of `Object` on its own and replace it with `bpy.types.Object`
```suggestion
from bpy.types import (
  Context, Material
)
```